utils/opencas.conf

```diff
@@ -5,12 +5,13 @@ version=19.3.0
 # of this file please refer to appropriate documentation
 
 # NOTES:
-# 1) It is highly recommended to specify cache/core device using path
-# that is constant across reboots - e.g. disk device links in
+# 1) It is limited to specify cache/core device using links in
```

Comment:
Suggest to rephrase:
It is required to specify ....

Reply:
thx